src/components/Modal.tsx

```
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import { Arrow, Clock, Close, Map_Pin, Plus, Users } from "../assets";
+
+interface ModalProps {
+  onChange?: string;
+}
+
+export const Modal = () => {
+  const [newTask, setNewTask] = useState<string>("");
+  const handleEnter = () => {};
```

ESLint

[error] 'handleEnter' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)